TCAT/Views/BusIcon.swift

```diff
@@ -9,12 +9,12 @@ import UIKit
 
 enum BusIconType: String {
 
-    case directionSmall, directionLarge, liveTracking
+    case directionSmall, directionLarge, lightSmallBlue, lightSmallRed, liveTracking
```

Comment:
If this is describing the color light blue/light red, then shouldn't the cases be `lightBlueSmall` or `smallLightBlue`?